games-puzzle/icebreaker/icebreaker-1.9.7.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A classic puzzle game"
+DESCRIPTION="So, uh, there's a bunch of penguins on an iceberg in \
+Antarctica. You have been selected to catch them so they can be shipped \
+to Finland, where they are essential to a secret plot for world domination."
+HOMEPAGE="https://mattdm.org/icebreaker/"
+COPYRIGHT="2000-2002 Matthew Miller"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://mattdm.org/icebreaker/1.9.x/icebreaker-1.9.7.tgz"
+CHECKSUM_SHA256="ca8b344a6a37b2041fba83271ae5e7566cbc8dc4c3367addd46f2a2f3573017e"
+GLOBAL_WRITABLE_FILES="settings/icebreaker.scores keep-old"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+
+PROVIDES="
+	icebreaker = $portVersion
+	app:IceBreaker = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl
+	lib:libsdl_mixer
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsdl
+	devel:libsdl_mixer
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	make prefix=$prefix bindir=$appsDir datadir=$dataDir \
+		highscoredir=$settingsDir mandir=$manDir
+}
+
+INSTALL()
+{
+	make prefix=$prefix bindir=$appsDir datadir=$dataDir \
+		highscoredir=$settingsDir mandir=$manDir install
+	mv $appsDir/icebreaker $appsDir/IceBreaker
+
+	addAppDeskbarSymlink $appsDir/IceBreaker IceBreaker
```

Comment:
Since **`/boot/system/apps`** is in the default PATH, I think it would be slightly better to keep $appsDir/icebreaker lowercase and only use IceBreaker in the AppDeskbar.
It would also allow to drop the **`mv $appsDir/icebreaker $appsDir/IceBreaker`**.
You would just need to change the next line to **`addAppDeskbarSymlink $appsDir/icebreaker IceBreaker`**.
But that's just my personal taste, and I don't even know what the best practices are for these kinds of questions. My rationale is that commands we type on a terminal are always lowercase. Yep, I know this is not a unix command ;-)
Anyway, please feel free to ignore this remark. I won't be cry if you do ;-)
To all: do not hesitate to tell what your prefer between $appsDir/icebraker and $appsDir/IceBraker, keeping in mind that in the Applications menu you'll see it as IceBreaker anyway.